push/ups_integration_with_openshift.md

```diff
@@ -0,0 +1,100 @@
+# Unified Push Server Integration with OpenShift 
+
+## Introduction
+
+This proposal outlines some developer experience issues with provisioning, setting up and configuring the Unified Push Server (UPS) on OpenShift, and integrating it with Apps using the Aerogear SDK.
+
+## Problem Description
+
+The Unified Push Server is made up of a Java application & a database (MySQL).
+It also has a dependecy on an auth provider (currently Keycloak) for users of the UPS Administration UI or REST API.
+There are docker images available for UPS.
+There is also a proof of concept UPS Ansible Playbook Bundle (APB) available.
+However, the APB just provisions UPS and MySQL currently.
+It expects an existing Keycloak instance to be running somewhere with the appropriate Realm, Client & Roles already setup.
```

Comment:
not really sure I fully understand what exactly the "first" problem is :-)

Reply:
See comment above where I mention I'll rephrase this section https://github.com/aerogear/proposals/pull/31#discussion_r172800370
What I'm trying to highlight here is the APB currently expects a lot of things to already exist i.e. Keycloak with specific relam & roles.
What we'll do is remove the need for this setup.
A User story that covers this (which I'll add to the doc) might be
'As a developer, I want UPS running in my OpenShift namespace, ready to go without any additional setup so I can start to use it immediately'